pkg/test/frameworkSettings/net/settings.targets

```diff
@@ -1,12 +1,14 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <!-- Desktop has cycles and dangling refs -->
```

Comment:
Why did you remove this?  The comment was meant to indicate why we can't run verify closure on the desktop assemblies.  I think we could actually fix that if we excluded the entire desktop targeting back from closure validation (or just the cycles).

Reply:
Oops I missread. I thought what was commented out was ShouldVerifyTypes and since we're setting it to true right below, that is why I deleted this. 

> I think we could actually fix that if we excluded the entire desktop targeting back from closure validation (or just the cycles).

https://github.com/dotnet/corefx/issues/29191

Reply:
Ha! I guess I am consistent.